### ISSUE TITLE
Fix default learning rate

### ISSUE THREAD

**Issue by sfluegel05** [ChEB-AI/python-chebai#150]
The default changed from 1e-3 to 1e-4 in [this commit](https://github.com/ChEB-AI/python-chebai/commit/2c159e8339b87d12db9cf94c418ba817c7ce3254#diff-286e6c43d041243386413f98ba9f93eca188d678d39b19a40450c57df5ab9979).

For training on ChEBI directly, 1e-3 seems to fare better than 1e-4. See [run with lr=1e-3](https://wandb.ai/chebai/chebai/runs/avqhai87/) and [run with lr=1e-4](https://wandb.ai/chebai/thesis-chebai/runs/xzx0f5o8/). 

## Todo
Change default to 1e-3 and add a comment that 1e-4 is better for OPT finetuning
